gds/tools/ioblitz/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+# set(CMAKE_BUILD_TYPE Release)
+#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE RelWithDebInfo)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_COLOR_MAKEFILE ON)
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+project(ioblitz LANGUAGES C CXX CUDA VERSION 0.3.0)
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
+
+find_package(CUDA REQUIRED)
+find_package(MPI REQUIRED)
+include_directories(${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+include_directories("/usr/local/gds/lib")
```

Comment:
The path names have changed to be 

/usr/local/cuda/targets/x86_64-linux/lib/
/usr/local/cuda/targets/x86_64-linux/include/